Assets/!/Scripts/Player/PlayerControllerV3.cs

```diff
@@ -7,8 +7,13 @@
 
 public class PlayerControllerV3 : MonoBehaviour, PlayerInputActions.IPlayerActions
 {
+    public bool isGamepad;
+    
+    public Vector2 moveDirection = Vector2.zero;
+    private Vector2 _pointerPosition;
+
+    public Vector3 moveDirectionVector3;
```

Comment:
Just a tip you cunt.

private class members should be camel case and have a prefix of _ e.g _pointerPosition.

public class members should be pascal case e.g MoveDirectionVector3

Reply:
Ah, thank you, noted and fixed across the PlayerController Script